docs/developers/docker.rst

```diff
@@ -0,0 +1,82 @@
+.. _docker:
+
+Docker
+======
+
+You can install a development and test environment using docker.
+
+
+.. _docker#assumptions:
+
+
+Assumptions
+-----------
+
+- You have docker installed and working on your system
+- The user you are using has permissions to use docker
+
+
+.. _docker#virtualenv:
+
+Create a virtualenv (optional)
```

Comment:
IMHO I wouldn't make it optional. When giving instructions it is better to reduce the number of optional things so when someone has to provide support it has less paths to explore.

Reply:
i think i agree - just that many seem to ignore this.

Reply:
Then whoever is providing support can tell them "We can't help you unless you follow the instructions which are well-tested and work."